contracts/sdk/gummyroll/src/types/index.ts

```diff
@@ -0,0 +1,762 @@
+// TODO(sorend): do we want to export the Anchor type in the SDK? We can modify all imports from contracts/target if so, but then we might need to custom hook into anchor build/test to keep this up to date.
```

Comment:
Ideally we would export the anchor type in the SDK, but this is tricky because anchor moves quickly as well.

But ideally yes

Reply:
let's take this out when we upgrade the Gummyroll SDK to Solita, some of Gummyroll's manual SDK methods depend on this and it's probably not worth refactoring since the next PR will overwrite the whole thing